pkg/remotes/docker/artifacts.go

```diff
@@ -0,0 +1,266 @@
+package docker
```

Comment:
What is the motivation of naming this docker? How about `artifacts` ?

Reply:
This serves a couple of purposes: 
1) to denote that this pkg is an extension/modification of remotes/docker in containerd
2) maintainers know which folder to look at in containerd
3) so that people who want to take a reference on this pkg know that it also depends on containerd/docker pkgs 

In summary, it's analagous to naming this windows.go or linux.go to denote os, but here I'm denoting flavor.